code/index.html

```diff
@@ -26,14 +26,17 @@ <h1>Does the person have</h1>
         <optgroup label="hair">
           <option value="brown">brown hair</option>
           <option value="yellow">yellow hair</option>
+          <option value="black">black hair</option>
           <!-- Add the rest of hair colors as options  -->
         </optgroup>
         <optgroup label="eyes">
           <option value="hidden">hidden eyes</option>
+          <option value="blueEyes">blue eyes</option>
           <!-- Add the rest of eye colors as options  -->
         </optgroup>
         <optgroup label="accessories">
           <option value="glasses">glasses</option>
+          <option value="hat">hat</option>
           <!-- Add the other accessory option here. (hat)  -->
         </optgroup>
         <optgroup label="other">
```

Comment:
I played a couple rounds and a few times wasn't able to filter it down to only one person. I think it would have been beneficial for the player to have more options so they can make a more qualified guess :)